src/h3/mod.rs

```diff
@@ -1925,11 +2045,10 @@ mod tests {
     }
 
     #[test]
+    // Client opens multiple control streams, which is forbidden.
```

Comment:
Other tests use `///` instead of `//`, same below. Don't really care which one is used, but we should be consistent.

Reply:
woops, I'll fix this up tomorrow

Reply:
fixed in the latest code review fixup commit